src/aleph/http/schema.clj

```diff
@@ -0,0 +1,46 @@
+(ns ^:no-doc aleph.http.schema
+  (:require [malli.core :as m]))
+
+(def server-port [:maybe :int])
+(def server-name [:maybe :string])
+(def remote-addr [:maybe :string])
+(def uri [:maybe :string])
+(def query-string [:maybe :string])
+(def scheme [:enum :http :https])
+(def request-method :keyword)
+(def content-type [:maybe [:or :string :keyword]])
+(def content-length [:maybe :int])
+(def character-encoding [:maybe :string])
```

Comment:
The `:maybe`s here are probably technically correct but I usually prefer to have only one way to express absence of a key (i.e. via `:optional true`). Otherwise, accidental passing of `nil` (e.g. due to spelling errors) easily slips through.

Reply:
I'm a bit concerned about breaking call sites if suddenly we forbid `nil` keys.

Reply:
Hm yeah, you're probably right, better to be conservative here, not worth releasing a breaking change for this.

Reply:
The schemas were 1-to-1 translations of the spec versions.